include/odp/api/spec/traffic_mngr.h

```diff
@@ -1918,6 +1951,29 @@ typedef struct {
 int odp_tm_node_fanin_info(odp_tm_node_t             tm_node,
 			   odp_tm_node_fanin_info_t *info);
 
+/** The odp_tm_node_stats_t record type  is used to return various stats
+ * supported by a given tm_node via the odp_tm_node_stats() function.
+ */
+typedef struct odp_tm_node_stats_t {
+	/** Node statistics. @see odp_tm_node_stats_type_t */
+	uint64_t stats[ODP_TM_NODE_STATS_MAX];
+} odp_tm_node_stats_t;
+
+/** Get tm_node stats
+ *
+ * The odp_tm_node_stats() function is used to extract runtime statistics
+ * associated with a given tm_node. The stats structure is written
+ * only on success.
+ *
+ * @param      tm_node    Specifies the tm_node to be queried.
+ * @param      clear      1: Clear stats before return.
+ *                        0: Don't clear stats.
```

Comment:
Do we really need the clearing? It easily causes synchronization overhead even when the application never actually requests clearing (the piece that writes the statistics has be be prepared for it).

Reply:
Ack, will remove.